src/lib/components/drs/formComponents/SubmitButton.js

```diff
@@ -95,6 +94,7 @@ const SubmitButton = props => {
                     props.setSuccessMessage(`Successfully created DrsObject with id: '${responseData.id}'`);
```

Comment:
the success message should be different for object creation vs update. Need to make this a parameter and set its value based on formViewType